README.md

```diff
@@ -1,139 +1,466 @@
-for more information about and examples of theming.
+To match Storybook to your site’s branding, change the colors in
+`.storybook/manager.js`. Any fonts can be added in
+`.storybook/manager-head.html`. See the [Storybook
+docs](https://storybook.js.org/docs/react/configure/theming) for more
+information about and examples of theming.
 
 ## Sass
-Sass can be compiled as part of the global styles.css file or to individual CSS files
-for use in a Drupal library.
 
-`@use` is used to import Sass variables, mixins, and/or functions into individual SCSS files. [`@import` is discouraged by the Sass team and will eventually be phased out.](https://sass-lang.com/documentation/at-rules/import).
-This means that most files will start with `@use 00-config/functions/gesso as *`. This allows you to use the
-design token accessor functions without an additional namespace. Other functions and mixins can be used similarly. Note that to avoid namespace
-collisions, only Gesso-related variables, mixins, and functions should be used with `*`.
+Sass can be compiled as part of the global styles.css file or to individual CSS
+files for use in a Drupal library.
 
-All Sass files that are compiled to individual CSS files must have a unique name,
-even if they are in different directories.
+`@use` is used to import Sass variables, mixins, and/or functions into
+individual SCSS files. [`@import` is discouraged by the Sass team and will
+eventually be phased out.](https://sass-lang.com/documentation/at-rules/import).
+This means that most files will start with
+`@use 00-config/functions/gesso as *`. This allows you to use the design token
+accessor functions without an additional namespace. Other functions and mixins
+can be used similarly. Note that to avoid namespace collisions, only
+Gesso-related variables, mixins, and functions should be used with `*`.
 
-### Global Styles
-Prefix the name of your Sass file with `_`, e.g. `_card.scss`. Add it to the appropriate
-aggregate file (i.e. `_components.scss`).
+All Sass files that are compiled to individual CSS files must have a unique
+filename, even if they are in different directories.
 
-### Individual Component/Library Styles
-DO NOT prefix the name of your Sass file with `_`, e.g. `menu.scss`. Import the config
-and global aggregate files. Import your SCSS file at the top of your Storybook file.
-See `menu.stories.jsx` for an example. Don't forget to add it to the `gesso.libraries.yml`
-file as well.
+### Global styles
+
+Prefix the name of your Sass file with `_`, e.g. `_card.scss`. Add it to the
+appropriate aggregate file (i.e. `_components.scss`).
+
+### Individual component/library styles
+
+DO NOT prefix the name of your Sass file with `_`, e.g. `menu.scss`. Import the
+config and global aggregate files. Import your SCSS file at the top of your
+Storybook file. See `menu.stories.jsx` for an example. Don’t forget to add it to
```

Comment:
Now that we broke out the menus, this should be `dropdown-menu.stories.jsx` (or any of the other menu examples)

Reply:
Done